src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -924,14 +924,28 @@ public class UpdateModelValueCommand : RecordableCommand
 
             #region Public Class Methods
 
-            public UpdateModelValueCommand(Guid modelGuid, string name, string value)
-                : this(new[] {modelGuid}, name, value)
+            /// <summary>
+            /// </summary>
+            /// <param name="modelGuid"></param>
+            /// <param name="name"></param>
+            /// <param name="value"></param>
+            /// <param name="workspaceModel">If workspace model is null, this command will be sent to the current workspace</param>
+            public UpdateModelValueCommand(Guid modelGuid, string name, string value, WorkspaceModel workspaceModel = null)
+                : this(new[] {modelGuid}, name, value, workspaceModel)
```

Comment:
@Benglin , this `modelGuid` is not guid of workspace model, but the guid of node model which will be updated, and previously `UpdateModelValueCommand` assumes the target workspace is current workspace.